apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -57,21 +47,10 @@ const ConfigScreen = () => {
   useEffect(() => {
     async function checkToken() {
       const tokenValid = await vercelClient.checkToken();
-
-      if (tokenValid) {
-        dispatchParameters({
-          type: actions.UPDATE_VERCEL_ACCESS_TOKEN_STATUS,
-          payload: true,
-        });
-      } else {
-        dispatchParameters({
-          type: actions.UPDATE_VERCEL_ACCESS_TOKEN_STATUS,
-          payload: false,
-        });
-      }
```

Comment:
I also removed this logic to update the status of the token within the installation parameters @matthew-gordon. Would appreciate any insight here that I might be overlooking. My approach was that the knowledge around if the token is valid or not does not need to be persisted in params, but rather can be handled in state on render.

Reply:
Mostly same answer above, this was just the mans to update based on an arbitrary status check to a `/tokens` endpoint in vercel and keep the ui up to date from early iterations, you are absolutely correct in your assumptions

Reply:
Perfect!